src/commons/index.js

```diff
@@ -27,5 +27,14 @@ module.exports = {
   },
   get withScrollReached() {
     return require('./withScrollReached').default;
+  },
```

Comment:
This one is not good.. 
There's a new temp file `commons/new.ts` (that will replace commons/index.js eventually) 
The typings are already exported there..

Reply:
Where is the call for the `commons/new.ts` file? I see only `commons/index.js`...

Reply:
We're using it in the components that need these typings.. 
View, Text, Button.. almost all of them..